ethexe/cli/src/chain_spec.rs

```diff
@@ -0,0 +1,30 @@
+use anyhow::Result;
+use ethexe_network::config::Multiaddr;
+use serde::Deserialize;
+use std::{fs, path::Path};
+
+#[derive(Default, Debug, Deserialize)]
+pub struct ChainSpec {
+    /// Address of Ethereum Router contract.
+    pub ethereum_router_address: String,
+
+    /// Addresses of Boot Nodes.
+    pub bootnodes: Vec<Multiaddr>,
+}
+
+pub fn testnet_config() -> ChainSpec {
+    ChainSpec {
+        ethereum_router_address: "0x05069E9045Ca0D2B72840c6A21C7bE588E02089A".to_string(),
```

Comment:
`testnet_config()` is used when the cli option `--chain-spec` is set to `dev`
```suggestion
        ethereum_router_address: "0xf90d2956B6F36194fbac181Fb95b2f67274821da".to_string(),
```

Reply:
Update it once merged, it's not a purpose of this PR
It's pure master from other repo